packages/appium/lib/cli/args.js

```diff
@@ -108,13 +108,12 @@ function makeInstallArgs(type) {
   return new Map([
     ...globalExtensionArgs,
     [
-      [type],
+      [`${type}Name`],
```

Comment:
would this change the actual argument behaviour?

Reply:
Hmm, seems like it. I guess I will revert this for now, but I'd still prefer to change it somehow - I just can't yet figure out where exactly it's used.